### PR TITLE
docs(readme.md): add link to Contentful community Slack

### PR DESCRIPTION
**Pull request by jacquesporveau**
# Purpose of PR

I've added a link to the Contentful community Slack where it is referenced in the README to avoid
newcomers having to guess where to find that and save them a google.

View the markdown here: https://github.com/jacquesporveau/forma-36/tree/jacquesporveau/update-readme#reach-out-to-us

## Also

Hey 👋 

I am new here. Trying to get some experience working on UI design libraries. I'll write some real code soon.

## PR Checklist

- [x] I have read the relevant `readme.md` file(s)
- [x] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components#commits)
- [x] Tests are added/updated/not required
- [x] Tests are passing
- [x] Storybook stories are added/updated/not required
- [x] Usage notes are added/updated/not required
- [x] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [x] Doesn't contain any sensitive information

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ contribute to a package, see the README of the corresponding package.
 - [Johannes Bugiel](https://github.com/wichniowski)
 - [Mike Mitchell](https://github.com/m10l)
 
-You can also reach out using the Contentful community Slack. We've setup a channel [#forma36](https://contentful-community.slack.com/messages/CFXGTMB98) in which we announce latest changes and updates.
+You can also reach out using the [Contentful community Slack](https://www.contentful.com/slack/). We've setup a channel [#forma36](https://contentful-community.slack.com/messages/CFXGTMB98) in which we announce latest changes and updates.
```
Comment (gui-santos) [inline README.md:75]: Wow! Thanks for that

I would just make a small suggestion, what do you think about moving this up? Like:

```
## Reach out to us

You can reach out to us using the [Contentful community Slack](https://www.contentful.com/slack/)...

### Maintainers
....
```

Comment (jacquesporveau) [reply README.md:75]: Yeah I think that makes sense. I guess the idea is that you would want people to reach out via the Slack channel before they went straight to a maintainer correct?

I'll push up another commit after work.

Comment (gui-santos) [reply README.md:75]: Yes... Probably we will add more communication channels here, but that's a good start I would say

**Comment by jacquesporveau** [comment]
Should be ready for another pass @gui-santos

**Comment by gui-santos** [comment]
@jacquesporveau Good job

**Pull request merged**